yacs/config.py

```diff
@@ -23,26 +23,32 @@
 import copy
 import io
 import logging
+import os
 from ast import literal_eval
 
 import yaml
 
+_PY2 = False
```

Comment:
nit: it would be nice to add a comment here to make it clear that py3 is used when this variable is not set

Reply:
Done